src/kbc_modules/cc_kbc/attester/tdx/mod.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2022 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use super::Attester;
+use anyhow::*;
+use serde::{Deserialize, Serialize};
+use std::path::Path;
+use tdx_attest_rs;
+
+const CCEL_PATH: &str = "/sys/firmware/acpi/tables/data/CCEL";
+
+pub fn detect_platform() -> bool {
+    Path::new("/dev/tdx-attest").exists()
+}
+
+#[derive(Serialize, Deserialize)]
+struct TdxEvidence {
+    // Base64 encoded CC Eventlog ACPI table
+    // refer to https://uefi.org/specs/ACPI/6.5/05_ACPI_Software_Programming_Model.html#cc-event-log-acpi-table.
+    cc_eventlog: Option<String>,
+    // Base64 encoded TD quote.
+    quote: String,
+}
+
+#[derive(Debug, Default)]
+pub struct TdxAttester {}
+
+impl Attester for TdxAttester {
+    fn get_evidence(&self, report_data: String) -> Result<String> {
+        let mut report_data_bin = base64::decode(report_data.clone())?;
```

Comment:
Fine for now, but in the future we might want to change `report_data` to be a `[u8; 48]` rather than a String.